packages/table/src/common/classes.ts

```diff
@@ -44,6 +45,8 @@ export const TABLE_OVERLAY_LAYER = "bp-table-overlay-layer";
 export const TABLE_POPOVER_WHITESPACE_NORMAL = "bp-table-popover-whitespace-normal";
 export const TABLE_POPOVER_WHITESPACE_PRE = "bp-table-popover-whitespace-pre";
 export const TABLE_REGION = "bp-table-region";
+export const TABLE_REORDERING = "bp-table-reordering";
+export const TABLE_REORDERING_CURSOR_SHIELD = "bp-table-reordering-cursor-shield";
```

Comment:
There are already `TABLE_OVERLAY` and `TABLE_OVERLAY_LAYER` constructs, but this isn't technically implemented as an overlay in the same sense, hence the new "shield" term. Happy to call it an overlay anyway though, if we're all cool with it.